packages/cli/src/commands/deviceprofiles/create.ts

```diff
@@ -169,17 +169,47 @@ export default class DeviceProfileCreateCommand extends APICommand {
 			createDeviceProfile, userInputProcessor(this))
 	}
 
-	protected async promptAndAddCapability(deviceProfile: DeviceProfileRequest, componentId: string, prompt = 'Capability ID'): Promise<void> {
-		const capabilityId = await chooseCapability(this, undefined, undefined, `${prompt}: `)
+	// TODO - update once capability versions are supported
+	protected async capabilityDefined(idStr:string): Promise<boolean> {
+		try {
+			const capability = await this.client.capabilities.get(idStr, 1)
+			return !!capability
+		} catch (e) {
+			return false
+		}
+	}
+	protected async promptAndAddCapability(deviceProfile: DeviceProfileRequest, componentId: string, prompt = 'Capability ID'): Promise<CapabilityId> {
+		let capabilityId: CapabilityId = {id: '', version: 0}
```

Comment:
We would normally have spaces inside the curly braces here too.

Reply:
Changed